tests/src/test/scala/org/http4s/headers/DNTSuite.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2013 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s
+package headers
+
+import org.scalacheck.Gen
+import org.scalacheck.Arbitrary
+import cats.syntax.either._
+
+class DNTSuite extends HeaderLaws {
+  implicit val dntGen = Gen.option(Gen.oneOf(true, false)).map(DNT(_))
+  implicit val arbDnt = Arbitrary[DNT](dntGen)
```

Comment:
We have an arbitraries object that includes some header arbitraries.  This (and the gen) might find a better home there.  If you put it in the `arbitraries` object instead of one of the traits, you'd avoid a bincompat mess vs. putting them in the (regretted) `ArbitraryInstances` trait.

Reply:
Build is failing because Scala 3 wants a type on the val.

Reply:
Ya, I added the type.